source/content.ts

```diff
@@ -113,6 +114,7 @@ import './features/hide-tips.css';
 import './features/hide-readme-header.css';
 import './features/hide-obvious-tooltips.css';
 import './features/clean-discussions.css';
+import './features/what-changed-since-previous-release.css';
```

Comment:
This should be inside the feature itself

Reply:
Is it needed? The styles are getting applied irrespective of it. Also, none of the current features has these CSS files imported.

Reply:
Yes it’s needed, the JS features should “load” their own CSS. Those in content.ts are CSS-only features

Look better, there are quite a few feature that load their own CSS